edx_proctoring/views.py

```diff
@@ -1450,6 +1454,47 @@ def put(self, request):
                 )
 
 
+class GroupedExamAllowancesByStudent(ProctoredAPIView):
+    """
+    Endpoint for the StudentProctoredGroupedExamAttemptsByCourse
+
+    Supports:
+        HTTP GET: return information about learners' allowances
+
+    **Expected Response**
+        HTTP GET:
+            The response will contain a dictionary with the allowances of a course grouped by student.
```

Comment:
#NIT can you provide an example response here? It makes the comments on this function much easier to read.